docs/guides/_toc.json

```diff
@@ -211,67 +201,52 @@
           "title": "Transpiler",
           "children": [
             {
-              "title": "Get started with the Qiskit transpiler",
-              "children": [
-                {
-                  "title": "Introduction to transpilation",
-                  "url": "/guides/transpile"
-                },
-                {
-                  "title": "Transpiler stages",
-                  "url": "/guides/transpiler-stages"
-                },
-                {
-                  "title": "Transpile with pass managers",
-                  "url": "/guides/transpile-with-pass-managers"
-                }
-              ]
+              "title": "Introduction to transpilation",
+              "url": "/guides/transpile"
             },
             {
-              "title": "Configure preset pass managers",
-              "children": [
-                {
-                  "title": "Default settings and configuration options",
-                  "url": "/guides/defaults-and-configuration-options"
-                },
-                {
-                  "title": "Set optimization level",
-                  "url": "/guides/set-optimization"
-                },
-                {
-                  "title": "Commonly used parameters for transpilation",
-                  "url": "/guides/common-parameters"
-                },
-                {
-                  "title": "Represent quantum computers",
-                  "url": "/guides/represent-quantum-computers"
-                }
-              ]
+              "title": "Transpiler stages",
+              "url": "/guides/transpiler-stages"
             },
             {
-              "title": "Advanced transpilation resources",
```

Comment:
I think this is still a useful subgroup. I'm not sure if there is a better name like "Custom transpilation and plugins"

Reply:
yeah im happy to keep this subfolder, I would just move it so it is the very last item (i.e. after the transpiler servcie)

Reply:
I'm restoring the subfolder - do we want to rename or keep the current name? @javabster

Reply:
i think keep the current name, "custom" could also imply custom passmanager stuff which isnt in this advanced section

Reply:
confirmed - keeping the current name. Subfolder is restored.